core/src/com/gaalf/game/precreatedEntities/balls/Ball.java

```diff
@@ -47,9 +46,9 @@ private SoundComponent addSoundComponent(Sound sound){
         return soundComponent;
     }
 
-    TextureComponent addTextureComponent(Texture texture){
+    TextureComponent addTextureComponent(Sprite sprite){
```

Comment:
It is kind of weird that it doesn't take in a texture object any longer considering the name of the function. But I see why you did it

Reply:
Also: maybe better to just take in the string to create the sprite as a parameter?

Reply:
Possibly change TextureComponent to SpriteComponent?

Reply:
Do you mean as in name of the function, or the actual components we are adding?

Reply:
Well both, we'd have a SpriteComponent and an addSpriteComponent

Reply:
I see now that the TextureComponent only has a sprite as a value, so I think it is a very good idea to rename yes :)